src/Apps/ArtAdvisor/04-Bio-Follows-Alerts/App.tsx

```diff
@@ -0,0 +1,128 @@
+import { FC, FormEvent, useEffect, useRef, useState } from "react"
+import {
+  Button,
+  Input,
+  Text,
+  Spacer,
+  Flex,
+  StackableBorderBox,
+  Box,
+  Spinner,
+  useToasts,
+} from "@artsy/palette"
+import { useSystemContext } from "System/SystemContext"
+import Markdown from "marked-react"
+
+type Message = {
+  role: string
+  content: string
+}
+
+export const App: FC = () => {
+  const [userInput, setUserInput] = useState<string>("")
+  const [messages, setMessages] = useState<Message[]>([])
+  const [isGenerating, setIsGenerating] = useState<boolean>(false)
+  const inputRef = useRef<HTMLInputElement>(null)
+
+  const { user } = useSystemContext()
+  const { sendToast } = useToasts()
+
+  useEffect(() => {
+    inputRef.current?.focus()
+  }, [])
+
+  const onSubmit = async (e: FormEvent) => {
+    e.preventDefault()
+    setIsGenerating(true)
+    try {
+      const res = await fetch("/api/advisor/4", {
+        method: "POST",
+        body: JSON.stringify([
+          ...messages,
+          { role: "user", content: userInput },
+        ]),
+        headers: {
+          "Content-Type": "application/json",
+          "X-ACCESS-TOKEN": user?.accessToken || "",
+        },
+      })
+
+      if (!res.ok) {
+        throw new Error("Failed response")
+      }
+
+      const parsedResponse = await res.json()
+
+      setMessages(parsedResponse)
+
+      if (inputRef.current) {
+        inputRef.current.value = ""
+        inputRef.current.focus()
+      }
+    } catch (e) {
+      sendToast({
+        variant: "error",
+        message:
+          "Something went wrong. Please try again. If you continue to see errors, message #product-ai-innovation-squad.",
```

Comment:
We may not want to leave the second part of this, but felt nicer than just repeatedly erroring.